testing/fio/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: wener <wenermail@gmail.com>
+# Maintainer: wener <wenermail@gmail.com>
+pkgname=fio
+pkgver=3.5
+pkgrel=0
+pkgdesc="Flexible I/O Tester"
+url="https://github.com/axboe/fio"
+arch="all"
+license="GPL-2.0"
+makedepends="linux-headers zlib-dev coreutils py-sphinx"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/axboe/fio/archive/fio-$pkgver.tar.gz"
+builddir="$srcdir/fio-$pkgname-$pkgver"
+options="!check"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+}
+doc() {
```

Comment:
`abuild` should package the subpackage automatically that way - without writing your own `doc()` function. Just specify - as already done - the subpackage in `subpackages` :+1: